OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs

```diff
@@ -25,9 +25,21 @@ public class WithInfantryBodyInfo : UpgradableTraitInfo, IRenderActorPreviewSpri
 
 		[SequenceReference] public readonly string MoveSequence = "run";
 		[SequenceReference] public readonly string AttackSequence = null;
+
+		[FieldLoader.LoadUsing("LoadWeaponSequences")]
+		[SequenceReference] public readonly Dictionary<string, string> AttackSequenceByArmament = new Dictionary<string, string>();
```

Comment:
My `[SequenceReference]` or rather the accompanying lint check isn't smart enough for dictionaries. Remove it and leave a `TODO` as I assume this won't be easy to solve.